### ISSUE TITLE
PHP Warning: Trying to access array offset on value of type bool in LinkHandlerTargetPageService.php

### ISSUE THREAD

**Issue by web-commerce-og** [georgringer/news#2019]
## Bug Report

**Current Behavior**
If you add a Linkhandler with the UserFunction LinkHandlerTargetPageService add to the page a link to a newsrecord with categories that don't have any detail pid safed. The Exception is thrown: 
```
(1/1) #1476107295 TYPO3\CMS\Core\Error\Exception
PHP Warning: Trying to access array offset on value of type bool in /var/www/html/private/typo3conf/ext/news/Classes/Service/LinkHandlerTargetPageService.php line 63
```

**Environment**
- TYPO3 version: 11.5.24
- news version: 10.0.3
- Is your TYPO3 installation set up with Composer (Composer Mode): yes

**Comment by georgringer** [comment]
wanna do a PR?

**Comment by georgringer** [comment]
fixed with 7b474e18
